core/src/main/java/org/apache/iceberg/actions/BaseExpireSnapshotsActionResult.java

```diff
@@ -18,6 +18,8 @@
  */
 package org.apache.iceberg.actions;
 
+/** @deprecated will be removed in 2.0.0. */
+@Deprecated
```

Comment:
if we're going to deprecate this, then this should mention that it will be removed in 1.3.0 (not 2.0.0)

Reply:
True. 
It is minor version removal as per [this](https://github.com/apache/iceberg/blob/3490db061c6adb300d0d0aaf27bae60c531359e1/CONTRIBUTING.md#semantic-versioning)

I followed some wrong example from the same module (https://github.com/apache/iceberg/blame/master/core/src/main/java/org/apache/iceberg/BaseMetadataTable.java#L111)

Reply:
done